packages/storage-plus/src/keys.rs

```diff
@@ -5,6 +5,33 @@ use crate::de::KeyDeserialize;
 use crate::helpers::namespaces_with_key;
 use crate::Endian;
 
+#[derive(Debug)]
+pub enum Key<'a> {
+    Ref(&'a [u8]),
+    Val8([u8; 1]),
+    Val16([u8; 2]),
+    Val32([u8; 4]),
+    Val64([u8; 8]),
```

Comment:
Actually, how about adding `Val128([u8; 16])` I am sure that sooner or later someone will want u128 as a key

Reply:
Didn't implement it for 128 bits on purpose. Enums are tagged unions. So, they have the size of their largest element (plus an extra for the tagging).

Adding 128 bits support would double the size of `Key`  to 16 bytes (plus the extra for tagging, which must end up in a 32-bits aligned boundary). So, 20 bytes instead of 12.

And, we are not using 128 int keys anywhere. We can easily add it, if you want. Or we can always add it later, when needed / requested.